content/applications/productivity/knowledge/articles_creation.rst

```diff
@@ -0,0 +1,243 @@
+
+.. tabs::
+   .. tab:: Source: Unsplash
+
+      Type your keywords in the search bar, then click on the chosen image. To generate more
+      results, click on :guilabel:`Load More` and load more pictures.
+   .. tab:: Source: URL
+
+      To add a picture by URL, click on :guilabel:`ADD URL` and type (or paste) the picture URL.
+   .. tab:: Source: Upload
+
+      To Upload an image directly from your computer, click on :guilabel:`UPLOAD AN IMAGE`, select
+      an image, and click on :guilabel:`open`.
+
+To remove the cover, hover over it to make the buttons appear and click on :guilabel:`Remove Cover`.
+
+.. note::
+
+   - To link Unsplash to your database, please refer to
+     :doc:`../../websites/website/optimize/unsplash`
+   - The articles are responsive, and so are the cover pictures. As a result, the images cannot
+     be repositioned manually to fit a particular screen, as they are automatically resized
+     depending on the device.
+
+Add an icon
+-----------
+
+To add an icon, hover above heading 1 and click :guilabel:`Add Icon`. Doing this sets a random emoji
+automatically. To change it, click it and select one from the emoji window. To remove it, proceed
+equally and click the red circled :guilabel:`x`.
+
+.. note::
+
+   - The emoji is also displayed before the corresponding article in the side panel hierarchic tree.
+   - Click the emoji on the side panel to change it without opening the related article.
+
+Use shortcuts
+=============
+
+To use a shortcut, type `/` and open the :doc:`powerbox <../../../developer/reference/frontend/odoo_editor>`.
+You can select from multiple shortcuts to insert blocks, images, files, etc. Some of them, such as
+`/Image` (to insert an image), `/Article` (to insert a link to an existing article), are common to
+all the apps. However, some shortcuts are inherent to the knowledge app and cannot be found or used
+in any other application.
+
+List of Knowledge-specific shortcuts
+------------------------------------
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Shortcuts
+     - Uses
+   * - :guilabel:`/Table of Content`
+     - Add a table of content with the article's headings.
+   * - :guilabel:`/Index`
+     - Show the first level of nested articles.
+   * - :guilabel:`/Outline`
+     - Show all nested articles.
+   * - :guilabel:`/Item Kanban`
+     - Insert a kanban view of article items.
+   * - :guilabel:`/Item List`
+     - Insert a list view of article items.
+   * - :guilabel:`/File`
+     - Embed a file that can be downloaded.
+   * - :guilabel:`/Template`
+     - Add a template section that can be inserted in messages, terms & conditions, or description
+       in other applications.
+
+Properties
+==========
+
+Properties are fields containing data and that can be added to articles by any user with
+:guilabel:`can write` access. These fields are shared between all the child articles or article
+items that are under the same parent.
```

Comment:
Properties are fields containing data that can be added to articles by any user with :guilabel:`can write` access. These fields are shared between all the child articles or article items under the same parent.